components/places/src/bookmark_sync/store.rs

```diff
@@ -147,13 +148,14 @@ impl<'a> BookmarksStore<'a> {
                 }
                 self.db.execute(&format!("
                     INSERT INTO mergedTree(localGuid, remoteGuid, mergedGuid, mergedParentGuid, level,
-                                           position, useRemote, shouldUpload)
+                                           position, useRemote, shouldUpload, mergedAt)
                     VALUES {}",
                     sql_support::repeat_display(chunk.len(), ",", |index, f| {
                         let d = &chunk[index];
-                        write!(f, "(?, ?, ?, ?, {}, {}, {}, {})",
+                        write!(f, "(?, ?, ?, ?, {}, {}, {}, {}, {})",
                             d.level, d.position, d.merged_node.merge_state.should_apply(),
-                            d.merged_node.merge_state.upload_reason() != UploadReason::None)
+                            d.merged_node.merge_state.upload_reason() != UploadReason::None,
+                            now.as_millis())
```

Comment:
is `as_millis()` required here (and below)? We've an `impl Display` for `Timestamp`. If we think this is better because it's more explicit, maybe we should have a followup to remove the `impl Display`? TBH though, I think I have a tiny preference for using `Display` because it screams "default repr for the DB" rather then needing the reader to think "oh, is that not using the default repr? <looks> Oh, it is, ok"